includes/admin/class-wc-admin-addons.php

```diff
@@ -491,6 +491,13 @@ public static function output_button( $url, $text, $theme, $plugin = '' ) {
 		$theme = __( 'Free', 'woocommerce' ) === $text ? 'addons-button-outline-green' : $theme;
 		$theme = is_plugin_active( $plugin ) ? 'addons-button-installed' : $theme;
 		$text  = is_plugin_active( $plugin ) ? __( 'Installed', 'woocommerce' ) : $text;
+		$url   = add_query_arg(
+			array(
+				'in-app-purchase-site'        => site_url(),
+				'in-app-purchase-woo-version' => WC_VERSION,
+			),
+			$url
+		);
```

Comment:
@haszari, perhaps, it worth to use just one method to do that, like [this one](https://github.com/woocommerce/woocommerce/pull/24075/commits/e470e5e563d76c41f2a0a25a546a484c0cc4abc2#diff-4812a6929a2f7c0db912c3bb7d367df7R490).

Reply:
Got it, it's in later commit. Sorry.